### DISCUSSION TITLE
avoiding obstacles

### DISCUSSION THREAD

**Discussion by Leon83858** [Courseplay/Courseplay_FS22#1808] [General]
feels like cp isnt that good in detecting fences or other obstacles anymore 🤔

**Comment by Leon83858** [comment]
![image](https://user-images.githubusercontent.com/23504996/181160425-5a188091-17ff-4f6f-9f94-49ed415596e4.png)
he thinks thats blocking him

**Comment by Leon83858** [comment]
![image](https://user-images.githubusercontent.com/23504996/181161128-2c8a3571-f5b5-443d-bfcd-25f5ca258a91.png)

**Reply by Leon83858** [reply]
not trying to steer more to turn in front of the fence

**Comment by Tensuko** [comment]
Bale is on the side, can't be whats blocking him.
If bale to clsoe in front of thim -> working as intended, we need some extra space.

He won't steer more becouse he follows the course -> working as intended.

**Reply by Leon83858** [reply]
fence: he was turning 
bale: moved a bit -> stopped -> moved a bit -> stopped

**Reply by Tensuko** [reply]
Just from that screenshot can't see whats happening.
Fence: Try Pathfinder turn.

**Reply by Leon83858** [reply]
all path finder stuff is on

**Reply by Leon83858** [reply]
i can imagine that the open gate might cunfuse him

**Reply by Tensuko** [reply]
Might be the reason, use more Headland will help as well.

We are aware of the sensor "problems" but I have it rather blocked then crash into something...
Might need some more adjustments.
